tbans/requests/fcm_request.py

```diff
@@ -83,17 +67,13 @@ def _fcm_message(self):
 
         # Add `notification_type` to data payload
         data_payload = self.notification.data_payload if self.notification.data_payload else {}
-        from consts.notification_type import NotificationType
+        from tba.consts.notification_type import NotificationType
         data_payload['notification_type'] = NotificationType.type_names[self.notification.__class__._type()]
 
+        # TODO: Add back in platform payloads
```

Comment:
- [ ] Add back in platform payloads

Still going back and forth with Firebase support on this one - absolutely wild

Reply:
Finally got this - sending an `apns` object in a request with only a `headers` object will cause the request to fail, for some reason. If we're going to pass an `apns` object, we need to send both a `headers` and a `payload` object, and the `payload` object needs to have an `aps` object setup. This seems like a bug in FCM - but we should look to work around it for now.